docs/sources/administration/plugin-management/index.md

```diff
@@ -17,9 +17,13 @@ weight: 600
 
 Besides the wide range of visualizations and data sources that are available immediately after you install Grafana, you can extend your Grafana experience with _plugins_.
 
-You can [install](#install-a-plugin) one of the plugins built by the Grafana community, or [build one yourself](/developers/plugin-tools).
+You can [install](#install-a-plugin) a wide range of the plugins that are built by the Grafana Labs, Partners, Community, or [build one yourself](/developers/plugin-tools).
 
-Grafana supports three types of plugins: [panels](/grafana/plugins?type=panel), [data sources](/plugins?type=datasource), and [apps](/grafana/plugins?type=app).
+Grafana supports three types of plugins:
+
+- [Panels](/grafana/plugins?type=panel)
+- [Data sources](/plugins?type=datasource)
+- [Apps](/grafana/plugins?type=app)
```

Comment:
```suggestion
- [Panels](/grafana/plugins/panel-plugins)
- [Data sources](/grafana/plugins/data-source-plugins)
- [Apps](/grafana/plugins/app-plugins)
```

Its odd that they were different, and also redirects so maybe better to go with where they actually land - (please verify this with a build of the docs)